version.gradle

```diff
@@ -25,7 +25,7 @@
  * as we want to manage the versions in a single source.
  */
 
-final def SPINE_VERSION = '1.5.6'
+final def SPINE_VERSION = '1.5.8'
```

Comment:
Please use version `1.5.7`.

Reply:
I need 1.5.8 because `base` and `core-java` (which is going to have this version soon too) need to have the same number so that `bootstrap` can be built.